### PR TITLE
Tag additional feats granting ancestral spells for Nephilim ancestry

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/feats/celestial-magic.json [hunk 1]
```diff
@@ -28,7 +28,48 @@
...
+                        "or": [
+                            "item:slug:clear-mind",
+                            "item:slug:sure-footing",
+                            "item:slug:share-life",
+                            "item:slug:revealing-light",
+                            "item:slug:humanoid-form",
+                            "item:slug:everlight"
+                        ]
+                    },
+                    "spellcasting:innate"
```
Comment (TikaelSol) [inline packs/feats/celestial-magic.json:47]: We should do a choice set for the spell. See my example in the data entry channel

Comment (ammalagonc) [reply packs/feats/celestial-magic.json]: Choice set added - please check when you get a moment.

Thanks!

File: packs/feats/celestial-magic.json [hunk 2]
```diff
@@ -28,7 +28,52 @@
...
+                "flag": "spell",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpeciciRule.Prompt.Spell"
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "{item|flags.pf2e.rulesSelections.spell}",
```
Comment (TikaelSol) [inline packs/feats/celestial-magic.json:59]:
```suggestion
                    "item:slug:{item|flags.pf2e.rulesSelections.spell}",
```

File: packs/feats/fiendish-magic.json [hunk 3]
```diff
@@ -28,7 +28,52 @@
...
+                "flag": "spell",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpeciciRule.Prompt.Spell"
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "{item|flags.pf2e.rulesSelections.spell}",
```
Comment (TikaelSol) [inline packs/feats/fiendish-magic.json:59]:
```suggestion
                    "item:slug:{item|flags.pf2e.rulesSelections.spell}",
```

**Pull request merged**